adp-helm-library/templates/_deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
         redeployOnChange: {{ randAlphaNum 5 | quote }}
     spec:
       priorityClassName: {{ .Values.deployment.priorityClassName| default "default" | quote }}
+      {{- if .Values.serviceAccount }}
```

Comment:
should this be an if statement?

Reply:
@DaRoszko yes, we need this for linting purpose on CI pipeline